### PR TITLE
GRPC length parameter for long sentences

### PR DESCRIPTION
**Pull request by j-bolivar-lt**
This configures a parameter called maxSentenceLength for long sentences to be handled in another way.

### CODE REVIEW

File: languagetool-core/src/main/java/org/languagetool/rules/GRPCRule.java [hunk 1]
```diff
@@ -72,18 +73,18 @@
...
-     messageBundle, configs, globalConfig, userConfig, motherTongue, altLanguages));
-     Rule exampleRule = GRPCRule.create(messageBundle,
-       RemoteRuleConfig.getRelevantConfig("EXAMPLE_ID", configs),
-      "EXAMPLE_ID", "example_rule_id",
-      Collections.singletonMap("example_match_id", "example_rule_message"));
-     rules.add(exampleRule);
-     return rules;
-   }
-  </pre>
+ <pre>
```
Comment (SteVio89) [inline languagetool-core/src/main/java/org/languagetool/rules/GRPCRule.java:76]: Please do not push whitespace changes

File: languagetool-core/src/main/java/org/languagetool/rules/GRPCRule.java [hunk 2]
```diff
@@ -48,6 +48,7 @@
 import com.google.common.collect.Streams;
 import com.google.common.util.concurrent.ListenableFuture;
 
+import com.sun.xml.bind.v2.TODO;
```
Comment (SteVio89) [inline languagetool-core/src/main/java/org/languagetool/rules/GRPCRule.java:51]: This should not be necessary.

Comment (fabrichter) [inline languagetool-core/src/main/java/org/languagetool/rules/GRPCRule.java:51]: Please delete this

File: languagetool-core/src/main/java/org/languagetool/rules/GRPCRule.java [hunk 3]
```diff
@@ -93,6 +94,9 @@ public abstract class GRPCRule extends RemoteRule {
   private static final int DEFAULT_BATCH_SIZE = 8;
   public static final String WHITESPACE_REGEX = "[\u00a0\u202f\ufeff\ufffd]";
   private static final String DEFAULT_DESCRIPTION = "INTERNAL - dynamically loaded rule supported by remote server";
+  /*TODO Delete this temporal fix as this is for speeding up execution for too long sentences*/
+  private final int maxSentenceLength = serviceConfiguration.getOptions().get("maxSentenceLength") == null ? Integer.MAX_VALUE : Integer.valueOf(serviceConfiguration.getOptions().get("maxSentenceLength"));
```
Comment (fabrichter) [inline languagetool-core/src/main/java/org/languagetool/rules/GRPCRule.java:98]: Please move the initialization to the constructor, where other options are initialized.

**Review by fabrichter — CHANGES_REQUESTED** [review]
Some small changes, then LGTM

**Review by fabrichter — APPROVED** [review]

**Pull request merged**